README.md

```diff
@@ -26,92 +26,92 @@ $ brew install --HEAD authzed/tap/zed
 
 ## Example Usage
 
-### Managing credentials
+### Logging into a permission system
 
-Configuring credentials is similar to [kubeconfig] in [kubectl].
-
-[kubeconfig]: https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/
-[kubectl]: https://kubernetes.io/docs/reference/kubectl/overview/
-
-API Tokens are stored in the system keychain.
+In order to interact with a permission system, zed first needs an API token.
+Zed stores API Tokens in the system keychain and all other non-sensitive data is stored in `$XDG_CONFIG_HOME/zed` falling back to `~/.zed`.
 
 ```
-$ zed config set-token jimmy@authzed.com tu_zed_hanazawa_deadbeefdeadbeefdeadbeefdeadbeef
-NAME             	ENDPOINT            	TOKEN
-jimmy@authzed.com	grpc.authzed.com:443	<redacted>
+$ zed token save exampledocs tu_zed_hanazawa_deadbeefdeadbeefdeadbeefdeadbeef
```

Comment:
I assume `exampledocs` is the permissions system here? I'd used something more descriptive like `my_permissions_system` or `example_permissions_system`